src/main/groovy/com/github/erdi/gradle/webdriver/task/ConfigureBinary.groovy

```diff
@@ -75,10 +75,13 @@ abstract class ConfigureBinary extends DefaultTask {
         def versionAndUri = new DriverUrlsConfiguration(driverUrlsConfiguration.get().asFile()).versionAndUriFor(downloadSpec())
         def installer = new DriverDistributionInstaller(project, downloadRoot.asFile.get(), driverName, versionAndUri)
         def distributionRoot = installer.getDistributionRoot(versionAndUri.version).get()
-        def osSpecificBinaryName = operatingSystem.getExecutableName(webDriverBinaryMetadata.binaryName)
+        def osSpecificBinaryNames = operatingSystem.getExecutableNames(webDriverBinaryMetadata.binaryName)
+
         def binaryFile = objectFactory.fileTree().tap {
             from(distributionRoot)
-            include("**/$osSpecificBinaryName")
+            osSpecificBinaryNames.each {
+                include(["**/${it}", "**/${it.toLowerCase()}"])
```

Comment:
I'm  sure there is a good reason, but why are you including `it.toLowerCase()`?

Reply:
Grolifant provides the file extensions in uppercase and that does not resolve to the files on Windows.  This ensures a match is found.

Reply:
added a comment